### DISCUSSION TITLE
Progress plan for Beve

### DISCUSSION THREAD

**Discussion by meftunca** [beve-org/beve#8] [Ideas]
Beve must now be ready for the community to play with him. By tailoring the organization to specific needs, we can start building the infrastructure for a large community.

1-> Manifesto, the main repository that hosts only documents with infrastructure and algorithms
2-> Port repos for supported languages
3-> Supported languages c/c++,rust,golang,JS/TS and matlab
4-> port repos will be responsible for their own testing and package release

**Comment by stephenberry** [comment]
I agree with this approach. I'm currently pulled in a couple of directions and focusing on low level coroutine support for Glaze, but I'll be working on this slowly. Feel free to start making new repositories for the respective language implementations of BEVE (c/c++,rust,golang,JS/TS and matlab)
